README.rst

```diff
@@ -3,22 +3,26 @@ Mozilla Product Details for Django
 
 |GHACI| |PyPI|
 
-**Mozilla Product Details** is a
-`library <http://viewvc.svn.mozilla.org/vc/libs/product-details/README?view=markup>`__
+**Product Details** is a public JSON API and a
+`repository <https://github.com/mozilla-releng/product-details/>`__
 containing information about the latest versions, localizations, etc. of
 `Mozilla <http://www.mozilla.org>`__ products (most notably Firefox,
 Firefox for mobile, and Thunderbird).
 
 From the original `README
-file <http://viewvc.svn.mozilla.org/vc/libs/product-details/README?view=markup>`__:
+file <https://github.com/mozilla-releng/product-details/blob/main/README.rst>`__:
 
 ::
 
-    This library holds information about the current builds of Firefox and
-    Thunderbird that Mozilla ships including:
+    Product Details is a public JSON API which contains release information
+    for Firefox (desktop and mobile) and Thunderbird. The data is managed by
+    the Release Management team through Ship-it and exported available at
+    this domain:
 
-    - Latest version numbers for all builds
-    - English and Native names for all languages we support
+    - htttps://product-details.mozilla.org/
```

Comment:
```suggestion
    - https://product-details.mozilla.org/
```

Reply:
Thanks for reading through it and keeping my Engrish in check;) Most of the content was taken from elsewhere so I'll double check the other sources too bc some things are pretty bizarre;D 

EDIT: yup this one is really [there](https://github.com/mozilla-releng/product-details/blob/03a4f3ada195a1c0caa81bfd0d08e60ade991658/README.rst?plain=1#L5) ;D :shrug: